### PR TITLE
Added new code

### PR DESCRIPTION
**Pull request by 23jodys**
# Description

Add the new modules we talked about

# Reviewer Checklist

## Code Review (Needs to SE reviews)

1. Verify that the code conforms to the design requirements. 
1. Verify that each code item is testable and traceable to the design requirement. 
1. Verify that the code is consistent internally and meets the coding standards identified in the software development plan.
1. Very that the complex expressions are identified and commented appropriately.

### CODE REVIEW

**Review by JWuSQA — APPROVED** [review]

**Review by JWuSQA — COMMENTED** [review]
All code review 👍

**Review by JWuSQA — APPROVED** [review]
All code review 👍

**Review by JWuSQA — CHANGES_REQUESTED** [review]
1,3,4 are 👍 but needs work on 2.

File: code/module1.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print "test"
+
```
Comment (JWuSQA) [inline code/module1.py]: This fails coding guidelines

**Review by JWuSQA — CHANGES_REQUESTED** [review]
Coding review 1,2,4 👍 but failing 3 for extra newline

**Comment by 23jodys** [comment]
@JWuSQA I fixed the coding guidelines.

**Review by JWuSQA — APPROVED** [review]
Coding review 2 is now 👍

**Pull request merged**